src/miner_consensus_mgr.erl

```diff
@@ -134,7 +133,19 @@ group_predicate("consensus" ++ _ = Name) ->
                     lager:debug("con pred ~p eh ~p h ~p", [Name, Height0, ElectionHeight]),
                     Height = list_to_integer(Height0),
                     Height < ElectionHeight;
-                [_Tag, Height0, _Hash] ->
+                _ ->
```

Comment:
Remind me why this changed?

Reply:
Are these just no longer possible?

Reply:
yeah, that was an ancient clause for backwards compatibility that I finally got rid of